engine/consensus/dkg/messaging_engine.go

```diff
@@ -13,6 +16,13 @@ import (
 	"github.com/onflow/flow-go/network"
 )
 
+// RETRY_MAX is the maximum number of times the engine will attempt to forward
+// a message
+const RETRY_MAX = 5
```

Comment:
We could use Go naming conventions and make this variable private, like `retryMax`